app/services/arcgis_api/mock/fixtures.rb

```diff
@@ -28,6 +28,38 @@ def self.request_candidates_error
       def self.load_response_fixture(filename)
         Rails.root.join('spec', 'fixtures', 'arcgis_responses', filename).read
       end
+
+      private_class_method def self.generate_suggestions(count = 5)
```

Comment:
May be cleaner to move all the methods into `class << self` ?

i.e.

```rb
module ArcgisApi
  module Mock
    module Fixtures
      class << self
        def request_suggestions_response
          # ...
        end

        # ...

        private

        def generate_suggestions(count = 5)
          # ...
        end
      end
    end
  end
end
```

Reply:
Alternatively, add a newline?
```suggestion
      private_class_method

      def self.generate_suggestions(count = 5)
```